packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -971,6 +971,11 @@ export abstract class FluidDataStoreContext
 			return;
 		}
 
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+		const originalStackTraceLimit = (Error as any).stackTraceLimit;
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+		(Error as any).stackTraceLimit = 30;
+
```

Comment:
Try removing the casting... it doesn't show red squiggles for me.

If you do need to cast, I think you can avoid a lot of the lint suppressions like this:
```suggestion
		const ErrorConfig = Error as unknown as { stackTraceLimit: number };
		const originalStackTraceLimit = ErrorConfig.stackTraceLimit;
		ErrorConfig.stackTraceLimit = 30;
```